### DISCUSSION TITLE
On the 100.64.0.0/10 undelegated range

### DISCUSSION THREAD

**Discussion by bcookatpcsd** [DNSCrypt/dnscrypt-proxy#2083] [Q&A]
less dnscrypt-proxy/plugin_block_undelegated.go

8< -- SNIP -- >8

        "10.in-addr.arpa",
        "100.100.in-addr.arpa",
        "100.51.198.in-addr.arpa",
        "101.100.in-addr.arpa",
        "102.100.in-addr.arpa",
        "103.100.in-addr.arpa",
        "104.100.in-addr.arpa",
        "105.100.in-addr.arpa",
        "106.100.in-addr.arpa",
        "107.100.in-addr.arpa",
        "108.100.in-addr.arpa",
        "109.100.in-addr.arpa",
        "110.100.in-addr.arpa",
        "111.100.in-addr.arpa",
        "112.100.in-addr.arpa",
        "113.0.203.in-addr.arpa",
        "113.100.in-addr.arpa",
        "114.100.in-addr.arpa",
        "115.100.in-addr.arpa",
        "116.100.in-addr.arpa",
        "117.100.in-addr.arpa",
        "118.100.in-addr.arpa",
        "119.100.in-addr.arpa",
        "120.100.in-addr.arpa",
        "121.100.in-addr.arpa",
        "122.100.in-addr.arpa",
        "123.100.in-addr.arpa",
        "124.100.in-addr.arpa",
        "125.100.in-addr.arpa",
        "126.100.in-addr.arpa",
        "127.100.in-addr.arpa",
        "127.in-addr.arpa",
        "16.172.in-addr.arpa",
        "168.192.in-addr.arpa",

8< -- SNIP -- >8

Is there an RFC range that describes that 100.100 - 100.127 range?

I found this on Wikipedia.. 

https://en.wikipedia.org/wiki/IPv4#Special-use_addresses

I also found this.. 

In BIND, empty reverse mapping zones for 100.64.0.0/16 till 100.127.0.0/16 (64 zones in total) are automatically created in the 'internal' view, if not configured otherwise.

I was looking to add more things, or possibly remove things.. is this a txt file tweak? or a recompile only?

Thanks in advance..

**Comment by jedisct1** [comment]
https://datatracker.ietf.org/doc/html/rfc6598

**Reply by jedisct1** [reply]
Changing the list requires a recompilation. But removing entries such as these is not going to be very useful. The same list is [blocked by servers](https://github.com/DNSCrypt/encrypted-dns-server/blob/master/undelegated.txt) since a query to authoritative servers wouldn't return anything valid either.
